ingestion/monitoring/Dockerfile

```diff
@@ -0,0 +1,71 @@
+# `python-base` sets up all our shared environment variables
+FROM python:3.9-slim as python-base
```

Comment:
3.10?

Reply:
I had problems installing poetry in the image when I based it on python:3.10-slim so I punted it. I will check again, it may have been an unrelated issue.

Reply:
Poetry [does not yet support Python 3.10](https://github.com/br3ndonland/inboard/pull/36)

Reply:
This appears to be the new way of installing poetry https://github.com/python-poetry/install.python-poetry.org -- I'm wary of the direct call to install.python-poetry.org, but should be possible to pin to a git tag

Reply:
Good catch. We were calling get-poetry from master too so we probably need to change this throughout the repo.